### PR TITLE
minor sort_values housekeeping

### PR DESCRIPTION
**Pull request by ryan-williams**
- [ ] ~Closes #xxxx~
- [ ] ~Tests added / passed~
- [x] Passes `black dask` / `flake8 dask`

a few mostly-cosmetic follow-ons to `sort_values` / #7286

### CODE REVIEW

File: dask/dataframe/core.py [hunk 1]
```diff
@@ -3962,15 +3962,15 @@ def select_dtypes(self, include=None, exclude=None):
         cs = self._meta.select_dtypes(include=include, exclude=exclude).columns
         return self[list(cs)]
 
-    def sort_values(self, other, npartitions=None, ascending=True, **kwargs):
+    def sort_values(self, by, npartitions=None, ascending=True, **kwargs):
```
Comment (jsignell) [inline dask/dataframe/core.py:3965]: You can just take the ascending=True out of here and out of `sort_values` below

**Review by jsignell — COMMENTED** [review]
This looks good, but while you are improving this I am wondering if we should do `@derived_from(pd.DataFrame)` instead.

**Comment by jsignell** [comment]
I'm going to go ahead and merge this actually, but @ryan-williams if you feel inspired to keep tweaking please go for it!

**Comment by ryan-williams** [comment]
thanks @jsignell, I'll try to do another PR for that (also will try hand at `Series.sort_values` if I have time)

**Pull request merged**